gitleaks.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: gitleaks
+  version: 8.18.2
+  epoch: 0
+  description: SAST tool for detecting and preventing hardcoded secrets like passwords, api keys, and tokens in git repos
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/gitleaks/gitleaks
+      expected-commit: ac4b5146b0f112df989b4374abb2b12799e37cba
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      # deps: github.com/BobuSumisu/aho-corasick@v1.0.3 github.com/charmbracelet/lipgloss@v0.5.0 github.com/fatih/semgroup@v1.2.0 github.com/gitleaks/go-gitdiff@v0.9.0 github.com/h2non/filetype@v1.1.3 github.com/rs/zerolog@v1.26.1 github.com/spf13/cobra@v1.2.1 github.com/spf13/viper@v1.8.1 github.com/stretchr/testify@v1.7.0 github.com/aymanbagabas/go-osc52/v2@v2.0.1 github.com/lucasb-eyer/go-colorful@v1.2.0 github.com/mattn/go-isatty@v0.0.17 github.com/mattn/go-runewidth@v0.0.14 github.com/muesli/reflow@v0.2.1-0.20210115123740-9e1d0d53df68 github.com/muesli/termenv@v0.15.1 github.com/rivo/uniseg@v0.2.0 github.com/davecgh/go-spew@v1.1.1 github.com/fsnotify/fsnotify@v1.4.9 github.com/hashicorp/hcl@v1.0.0 github.com/inconshreveable/mousetrap@v1.0.0 github.com/lucasjones/reggen@v0.0.0-20200904144131-37ba4fa293bb github.com/magiconair/properties@v1.8.5 github.com/mitchellh/mapstructure@v1.4.1 github.com/pelletier/go-toml@v1.9.3 github.com/pmezard/go-difflib@v1.0.0 github.com/spf13/afero@v1.6.0 github.com/spf13/cast@v1.3.1 github.com/spf13/jwalterweatherman@v1.1.0 github.com/spf13/pflag@v1.0.5 github.com/subosito/gotenv@v1.2.0 golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c golang.org/x/sys@v0.6.0 golang.org/x/text@v0.3.8 gopkg.in/ini.v1@v1.62.0 gopkg.in/yaml.v2@v2.4.0 gopkg.in/yaml.v3@v3.0.0-20210107192922-496545a6307b
```

Comment:
uncomment this maybe? :)

Reply:
sure, i updated in new commit. i have tested on apko also and add test steps :#